src/core/Decorations.ts

```diff
@@ -131,6 +190,42 @@ export default class Decorations {
     );
   }
 
+  private constructHatStyleMap() {
+    const hatStyleSettings = vscode.workspace
+      .getConfiguration("cursorless")
+      .get<HatStyleSettingMap>("hatStyles")!;
+
+    hatStyleSettings.default = { penalty: 0, enabled: true };
```

Comment:
Do we want to allow user to configure default color and shape `penalty` / `enabled`?  If so, note that settings setup will have to change because they'll both want to use the setting `cursorless.hatStyles.default`